apps/test/unit/templates/SchoolAutocompleteDropdownTest.js

```diff
@@ -1,15 +1,102 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 import {expect} from '../../util/configuredChai';
+import sinon from 'sinon';
 import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown';
 
 describe('SchoolAutocompleteDropdown', () => {
-  it('renders VirtualizedSelect', () => {
-    const wrapper = shallow(
+  let schoolAutocompleteDropdown;
+  let handleChange;
+  let select;
+  beforeEach(() => {
+    handleChange = sinon.spy();
+    schoolAutocompleteDropdown = shallow(
       <SchoolAutocompleteDropdown
-        showErrorMsg={false}
+        value="12345"
+        onChange={handleChange}
       />
     );
-    expect(wrapper.find('VirtualizedSelect').exists()).to.be.true;
+
+    select = schoolAutocompleteDropdown.find('VirtualizedSelect');
+  });
+
+  it('renders VirtualizedSelect', () => {
+    expect(select.exists()).to.be.true;
```

Comment:
We use [`chai-enzyme`](https://github.com/producthunt/chai-enzyme) which lets you write:
```js
expect(select).to.exist;
```
[(docs)](https://github.com/producthunt/chai-enzyme#present)